### ISSUE TITLE
Bitcoin Core Daemon on OSX Yosemite

### ISSUE THREAD

**Issue by kevcooper** [bitcoin-dot-org/bitcoin.org#1026]
I am currently running a full node (v0.11.0) on OS X Yosemite (10.10.5) and would be happy to contribute a short tutorial and some screenshots in addition to whatever else is needed!

**Comment by harding** [comment]
@kevcooper thanks!  Just earlier today someone else said that they were going to start working on that in issue #1023.  What I suggest is that you subscribe to that issue (it's in the right-side menu when you visit that issue) and see what happens; if there's no activity for a week or so, I'll post a comment and you can take over.

Let me know if that works for you.  Thanks again!

**Comment by kevcooper** [comment]
sounds good!

**Comment by drazisil** [comment]
@kevcooper it looks like the person from #1023 isn't going to write the instructions, as they appear to be no longer running bitcoind, so you can probably go ahead.

**Comment by kevcooper** [comment]
Okay I'll get started!

**Comment by kevcooper** [comment]
I've got a rough draft done [here](https://github.com/kevcooper/bitcoin.org/blob/osx-daemon/en/full-node.md#bitcoin-core-daemon-osx-daemon). Let me know what I need to change or add. I didn't include any screenshots since the entire process is done from the command line, but again I can add some if necessary. Also, I know I shouldn't link to my personal gist for the Launch Agent in the final version but I wasn't quite sure where else to put it so let me know where on the site it should go.

**Comment by carnesen** [comment]
Today I'll review the draft and try the Launch Agent instructions. I know @harding and @saivann aren't Mac folks.

**Comment by carnesen** [comment]
I've read through the draft, and it looks pretty good! I do have some feedback though. Perhaps you could submit a pull request upon which we'll provide line comments? @harding @saivann, please correct me if that's not the preferred workflow here.

Speaking of which I think the right place for that plist is [the Bitcoin Core repo alongside the config files for upstart, systemd, etc.](https://github.com/bitcoin/bitcoin/tree/0.11/contrib/init). That'd be a separate pull request, of course.

**Comment by saivann** [comment]
@kevcooper Agreed with @carnesen, if you could submit your osx-daemon branch as a pull request, that'd be great. Thanks for your help!

**Comment by kevcooper** [comment]
Okay I'll squash and submit a PR later today. I also submitted a [PR](https://github.com/bitcoin/bitcoin/pull/6621) to Bitcoin core for the launch agent so that'll need to be merged before I can use it in the guide.

**Comment by carnesen** [comment]
Pull was submitted as https://github.com/bitcoin-dot-org/bitcoin.org/pull/1046

@kevcooper if you include the phrase "closes #1026" in your commit message, GitHub will close this issue automagically when the pull gets merged into master.

**Comment by kevcooper** [comment]
@carnesen awesome I'll add that in my next commit!
